aws/policy/application-services.yaml

```diff
@@ -107,12 +107,15 @@ Statement:
       - kinesis:RemoveTagsFromStream
       - kinesis:StartStreamEncryption
       - kinesis:StopStreamEncryption
+      - glue:DeleteCrawler
       - glue:DeleteJob
+      - glue:GetCrawler
```

Comment:
We also need glue:GetCrawlers, for use by the terminator class.  This needs to operate globally, so it should go under `  - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`

Reply:
@jillr Thank you for reviewing this - that's an interesting point.

I see that there are no other Glue-related permissions under `Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`.

If they are in fact needed for terminator class, should I add permissions for Glue crawlers, jobs, and connections:

```
- glue:GetConnections
- glue:GetCrawlers
- glue:GetJobs
```

What do you think?

Reply:
@ichekaldin Yes please, thanks!

Reply:
@jillr Done